modules/conflict_resolver/README.md

```diff
@@ -0,0 +1,65 @@
+# Conflict Resolver
+
+## Purpose
+
+The conflict resolver is intended to allow the resolution of conflicts
+(different data entry for a given field between the "single data
+entry" and "double data entry" variations) for instruments which
+have double data entry enabled.
+
+## Intended Users
+
+The conflict resolver is usually used by data entry staff to resolve
+double data entry conflicts.
+
+Some studies may have protocols which involve supervisors or committees
+deciding how to properly resolve conflicts, and different studies may
+have different policies of if the same or different data entry staff
```

Comment:
I don't understand this part of the sentence: "different studies may have different policies **of if** the same or different data entry staff is responsible for conflict resolution, but such policies are not enforced by LORIS."
Is there a chance it could be clarified?

Reply:
It was a sentence I added to try and address comments from @me-evans's review (it's hidden above in the GitHub history if you want to read it). What's unclear about it? I can change it to "of whether", but I don't think that address the clarity of the sentence much..

Reply:
I think I am starting to understand the sentence. Maybe it could be rephrase like this (if I understood it correctly):
"Some studies may have protocols which involve supervisors or committees deciding how to properly resolve conflicts. For example, studies may adopt different policies regarding the personnel that should be responsible for conflict resolution (could be the same data entry staff that performed the double data entry or a different data entry staff). Such policies are not enforced by LORIS."

Reply:
Sounds good to me. I changed it.